manifests/secondarydns.yaml

```diff
@@ -71,6 +71,8 @@ spec:
     metadata:
       labels:
         k8s-app: secondary-dns
+      annotations:
+        kubectl.kubernetes.io/default-logs-container: status-monitor
```

Comment:
Maybe even - `kubectl.kubernetes.io/default-container. What do you think?

Reply:
will check if default-container can stand for both

Reply:
it seems it will be supported and replace it on 1.25
but atm it doesnt work yet with 1.24 (i tried)

https://github.com/kubernetes/enhancements/issues/2227

Reply:
we have it on virt-launcher

Reply:
virt-launcher doesnt have 2 containers right ?
i tried locally with the tag and logs didnt work, please try if you want

Reply:
virt-launcher has more than two. I will try.

Reply:
I tested it locally, `kubectl.kubernetes.io/default-container` works for me while `kubectl.kubernetes.io/default-logs-container` doesn't.

Reply:
which kubectl do you have, can you do paste the full version ?
which provider do you run ?  the one we have in this repo  1.24.7 ?

Reply:
`kubectl version --short
Flag --short has been deprecated, and will be removed in the future. The --short output will become the default.
Client Version: v1.25.3
Kustomize Version: v4.5.7
Server Version: v1.24.7
`